test/conftest.py

```diff
@@ -130,6 +130,10 @@ def setup_pg():
             # don't need to wait for autovacuum workers when shutting down
             "autovacuum": "off",
         })
+        # Setting name has changed in PG 13, set comparison needed because 9.6 is somehow larger than 13 for the comparison
+        if db.pgver >= "13" and db.pgver not in {"9.5", "9.6"}:
```

Comment:
LooseVersion would be better (and I guess this same thing affects pglookout, which now doesn't set the max wal for 9.5/9.6 in tests) but as this is only test code and should have effectively the same result I'll merge.